docs/deployment/contract-deployment-options.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 1.4
+title: Contract Deployment Options
+---
+
+When using `LSPFactory` you have the ability to specify custom deployment parameters to be applied to your contracts.
+
+The `deploy` function takes an object `contractDeploymentOptions` as its second parameter where the contract `version`, `byteCode` and `libAddress` can be specified:
+
+> `contractDeploymentOptions?` - `Object`:
+
+- `version?` - `string`: The Contract version you want to deploy. Defaults to latest version of [lsp-smart-contracts](https://github.com/lukso-network/lsp-smart-contracts) library.
+- `byteCode?` - `string`: Custom bytecode to be deployed.
+- `deployProxy?` - `boolean`: Whether the contract should be deployed using a proxy contract implementation (eg: [EIP1167](https://eips.ethereum.org/EIPS/eip-1167)). Defaults to true.
+- `libAddress?` - `string`: The Address of a Base Contract to be used in deployment as implementation behind a proxy contract (eg: [EIP1167](https://eips.ethereum.org/EIPS/eip-1167)).
+
+
+This is an optional parameter and so may be ommitted. If no contract deployment options are specified, `LSPFactory` will deploy a proxy contract which defers to a base contract implementation as per the [EIP1167](https://eips.ethereum.org/EIPS/eip-1167) standard.
```

Comment:
~~ommitted~~ **omitted**

Reply:
Thanks, including these changes in next PR